command/agent/config.go

```diff
@@ -286,8 +286,15 @@ type ClientConfig struct {
 	// the host
 	BridgeNetworkSubnet string `hcl:"bridge_network_subnet"`
 
+	// HostNetworks describes the different host networks available to the host
+	// if the host uses multiple interfaces
 	HostNetworks []*structs.ClientHostNetworkConfig `hcl:"host_network"`
 
+	// BindWildcardDefaultHostNetwork toggles if whe there are no host networks,
```

Comment:
This comment or the one above would make good docs.

```suggestion
	// BindWildcardDefaultHostNetwork toggles if when there are no host networks,
```